src/renderer/browser/components/RequestsProcessor/ClaimGas/index.js

```diff
@@ -14,7 +14,7 @@ import withRejectMessage from '../../../hocs/withRejectMessage';
 const mapAuthDataToProps = ({ address, wif }) => ({ address, wif });
 const mapSendDataToProps = (txid) => ({ txid });
 
-export default function makeClaimComponent(claimActions) {
+export default function makeGetClaim(claimActions) {
```

Comment:
Did you mean "makeGasClaim" or is this name intentional? xD

Reply:
Whoops, meant for this to be `makeClaimGas`.  The format is supposed to be `make<ComponentName>`.  Thanks!